cmake/api_macros.cmake

```diff
@@ -86,6 +86,20 @@ macro(_OPENDDS_GENERATE_EXPORT_MACRO_COMMAND  target  output)
   set(${output} ${_output_file})
 endmacro()
 
+macro(OPENDDS_MARK_AS_GENERATED target)
```

Comment:
Please add to documentation

Reply:
I think with the new changes this is no longer necessary as there is no new macro, and the macro is built in to OpenDDS, and thus no longer requires the user to call anything manually.